lib/phi_attrs/phi_record.rb

```diff
@@ -356,6 +358,28 @@ def class_phi_context
       self.class.__phi_stack[-1]
     end
 
+    # The unique identifiers for everything with access allowed on this instance.
+    #
+    # @private
+    #
+    # @return Array[String] all the user_id's passed in to allow_phi!
+    #
+    def all_phi_allowed_by
+      all_phi_context.map { |c| c[:user_id] }.to_s
```

Comment:
this says it's supposed to return an `Array[String]`, but it looks like it actually returns a String (of an array). Did you mean:

```suggestion
  all_phi_context.map { |c| c[:user_id].to_s }
```

Reply:
No should be a string, I pushed up a change with some additional formatting fixes to the string